src/UI/templates/default/Input/Rating/tpl.rating.html

```diff
@@ -0,0 +1,32 @@
+
+<div class="input-group">
+
+	<span class="il-input-rating-topic">{TOPIC} </span>
```

Comment:
Despite my general concern with the hideTopic thingy, with this template there is "topic"-clutter in the html even if there is no `{TOPIC}`.